README.md

```diff
@@ -168,8 +168,7 @@ spec:
             - containerPort: 2575
               protocol: TCP
               name: "port"
-          command:
```

Comment:
I think it makes sense to format Upgrade guide by version. So, old content goes into 0.1.1 section, this here goes into 0.2.0 (presumed), etc.

Reply:
Yeah that makes sense. Btw, can you please change the container name here? Maybe to 0.2.0 since this won't work with the existing container in the readme (0.1.14)

Reply:
Updated upgrade guide and versions in gradle files from 0.1 to 0.2.

Reply:
I also mean in this readme file. Here we're using version 0.1.14 which won't work with the args command

Reply:
Right, changed version in Readme examples to 0.2.0 as well.